addons/tech/optic_sxmr10/CfgWeapons.inc

```diff
@@ -0,0 +1,77 @@
+#define CLASSES_WEAPONS \
+  "SWS_Optic_SxMR10"
+
+class CfgWeapons {
+  class optic_AMS_base;
+  class SWS_Optic_SxMR10: optic_AMS_base {
+    ITEM_META(2);
+    displayName = QNAME(SxMR-10);
+		picture = "\x\sws\addons\tech\optic_sxmr10\data\icon_optic_sxmr10_ca.paa"; // I wanted to use a QSUBPATHTOF here but it kept adding .sqf. TODO: Diagnose this later?
+    descriptionShort = QUOTE(Experimental optic based on the US Optics MR-10.<br/> A night vision mode has been added.);
+
+    ACE_ScopeHeightAboveRail = 3.8933;
```

Comment:
So, I did try to do that at first but ran into some very strange bugs. My initial assumption was that optics are haunted, because it's Arma. I'll do a run today of pruning them back and seeing which ones properly stick in config.